manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -371,6 +372,68 @@ spec:
           defaultMode: 420
           secretName: gatekeeper-webhook-server-cert
 ---
+kind: Pod
+apiVersion: v1
+metadata:
+  labels:
+    control-plane: controller-manager
+    gatekeeper.sh/system: "yes"
+  name: gatekeeper-audit
+  namespace: gatekeeper-system
+spec:
+  containers:
+  - args:
+    - --audit-interval=30
+    - --operation=audit
+    - --logtostderr
+    command:
+    - /manager
+    env:
+    - name: POD_NAMESPACE
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.namespace
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+            fieldPath: metadata.name
+    image: quay.io/open-policy-agent/gatekeeper:v3.1.0-beta.7
+    imagePullPolicy: Always
+    livenessProbe:
+      httpGet:
+        path: /healthz
+        port: 9090
+    name: audit
+    ports:
+    - containerPort: 8888
+      name: metrics
+      protocol: TCP
+    - containerPort: 9090
+      name: healthz
+      protocol: TCP
+    readinessProbe:
+      httpGet:
+        path: /readyz
+        port: 9090
+    resources:
+      limits:
+        cpu: 1000m
+        memory: 512Mi
```

Comment:
i wonder if these limits should be changed here or in webhook one

Reply:
Changed in what sense?

I would expect roughly equal CPU/RAM usage between the two, as audit may be more bursty, but should hit similar peaks, though having some real load data would make this clearer.